src/components/nano/ModalConfirmUnregister.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from 'react';
+import { t } from 'ttag';
+import $ from 'jquery';
+
+export default function ModalConfirmUnregisterNanoContract({ onClose, ncId, success }) {
+  useEffect(() => {
+    $('#modalConfirmUnregister').modal('show');
```

Comment:
suggestion(non-blocking): Extract `#modalConfirmUnregister` to a constant on the file level. Reference from `ModalUnhandledError`:

https://github.com/HathorNetwork/hathor-wallet/blob/a968e6747afb600bb8bf9ed1b83214ef608cbdaa/src/components/ModalUnhandledError.js#L16-L17

Reply:
Done, thanks! https://github.com/HathorNetwork/hathor-wallet/pull/625/commits/1eb97caf38dffe6a3feafdf94db3825369f6e550